ouroboros-consensus/src/Ouroboros/Storage/ChainDB/Impl/Background.hs

```diff
@@ -186,12 +186,29 @@ copyToImmDB CDB{..} = withCopyLock $ do
     mustBeUnlocked = fromMaybe
                    $ error "copyToImmDB running concurrently with itself"
 
--- | Watches the current chain for changes. Whenever the chain is longer than
--- @k@, then the headers older than @k@ are copied from the VolatileDB to the
--- ImmutableDB (with 'copyToImmDB'). Afterwards, a garbage collection of the
--- VolatileDB is scheduled using ('scheduleGC') for the 'SlotNo' of the most
--- recent block that was copied.
-copyToImmDBRunner
+-- | Copy blocks from the vol DB to imm DB and take snapshots of the ledger DB
```

Comment:
This docstring contains a mix of "vol DB", "VolatileDB" and "volatile DB" (same for the other two). Can you make it consistent and use either "VolatileDB" or "VolDB"?